packages/esm-service-queues-app/src/add-provider-queue-room/add-provider-queue-room.resource.ts

```diff
@@ -74,3 +78,31 @@ export function useProvidersQueueRoom(providerUuid: string) {
     mutate,
   };
 }
+
```

Comment:
Is the below something that would typically be found in a resource?

Reply:
Yes, it's a typical pattern to define react hooks within *resource.ts. I try to consolidate hooks that are used in multiple places in hooks.ts, but I don't foresee this being used in more than one place.